ehc_assignment_2.rb

```diff
@@ -0,0 +1,39 @@
+#! usre/bin/ruby
+require "net/http"
+require "uri"
+class EhcAssignment 
+  def crawl(date)
+    url = "https://services.ecourts.gov.in/ecourtindiaHC/cases/highcourt_causelist_qry.php"
+    request = Net::HTTP.post_form(URI(url),{"action_code" => "pulishedCauselist", "causelist_dt" => date,"state_code" => "1","dist_code" => "1","court_code" => "2"})
```

Comment:
what if this throw error like if don't have internet connection or site is not reachable how will you handle this.?